### PR TITLE
Github Actions

### LINKED ISSUE #5: Set up GitHub Actions
It would be nice to have linting/typing/testing run automatically

### PR DESCRIPTION
**Pull request by AetherUnbound**
This PR adds GitHub Actions! Resolves #5

### CODE REVIEW

File: tests/test_signal_scanner_bot.py [hunk 1]
```diff
@@ -1,37 +1,5 @@
...
-    """Test the CLI."""
-    runner = CliRunner()
-    result = runner.invoke(cli.main)
-    assert result.exit_code == 0
-    assert "signal_scanner_bot.cli.main" in result.output
-    help_result = runner.invoke(cli.main, ["--help"])
-    assert help_result.exit_code == 0
-    assert "--help  Show this message and exit." in help_result.output
+def test_dummy():
+    assert True
```
Comment (JacksonMaxfield) [inline tests/test_signal_scanner_bot.py:5]: HUGE.

**Comment by JacksonMaxfield** [comment]
I know there are methods for "testing" a Docker image but it could also be good to build the docker image as a gh-action just to make sure everything there builds properly.

File: tests/test_signal_scanner_bot.py (hunk 1 above)
Comment (AetherUnbound) [reply tests/test_signal_scanner_bot.py:5]: img

**Comment by AetherUnbound** [comment]
> I know there are methods for "testing" a Docker image but it could also be good to build the docker image as a gh-action just to make sure everything there builds properly.

I knew I was missing something! Thanks, I'll add this too 👍

**Pull request merged**